pkg/osgi_config_manager.go

```diff
@@ -105,7 +105,7 @@ func (cm *OSGiConfigManager) FindByFactory(fpid string, cid string) (*osgi.Confi
 			if err != nil {
 				return nil, err
 			}
-			log.Infof("%s > found config '%s' with CID '%s'", cm.instance.ID(), pid.ID, config.CID())
+			log.Debugf("%s > found config '%s' with CID '%s'", cm.instance.ID(), pid.ID, config.CID())
```

Comment:
please check if other `Find` methods for bundle/components etc are having such logging. if not.. delete it